hstream-store/HStream/Store/Stream.hs

```diff
@@ -211,7 +212,7 @@ updateGloLogPathCache streamid key logid =
       Nothing -> Cache.insert c streamid (Map.singleton key logid) >> pure c
       Just mp -> Cache.insert c streamid (Map.insert key logid mp) >> pure c
 
-newtype ArchivedStream = ArchivedStream CBytes
+newtype ArchivedStream = ArchivedStream { getArchivedStreamName :: CBytes}
```

Comment:
For the api design, `getArchivedStreamName` should not be exported.

Reply:
But subscription need this information to resume consumption